server/events/yaml/raw/global_cfg.go

```diff
@@ -85,6 +85,19 @@ func (g GlobalCfg) Validate() error {
 
 func (g GlobalCfg) ToValid(defaultCfg valid.GlobalCfg) valid.GlobalCfg {
 	workflows := make(map[string]valid.Workflow)
+
+	applyReqs := defaultCfg.Repos[0].ApplyRequirements
```

Comment:
Let's maybe put a comment here noting that we always initialize global config with at least 1 repo.